script/run-unit-test.sh

```diff
@@ -14,7 +14,7 @@ if [[ -z "$1" ]] || [[ $1 == -* ]]; then
   exit $?
 fi
 
-BABEL_ENV=test $MOCHA src/platform/testing/unit/helper.js "$@"
+BABEL_ENV=test $MOCHA --opts src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js "$@"
```

Comment:
Does this maybe warrant its own PR?

Reply:
Oh, yup. That wasn't supposed to sneak in here.